### PR TITLE
Fixes #1

### LINKED ISSUE #1: Error when contents is empty
Use case:

Yeoman generator is trying to copy an image from path XXX to YYY. YYY is within the project folder and the conflict should be now resolved. 

Unfortunately, due to the fact that contents is null, buffer throws an error:

``` js
this.length = +subject.length > 0 ? Math.ceil(subject.length) : 0;
TypeError: Cannot read property 'length' of null
    at new Buffer (buffer.js:184:31)
```

Conflict should always be raised if there's something at filePath and new content is null.

### PR DESCRIPTION
**Pull request by grabbou**
Added test case + if statement as described in #1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.idea/
```
Comment (SBoudrias) [inline .gitignore:2]: Don't modify this file.

This should go in your own global .gitignore

**Comment by SBoudrias** [comment]
I'm not sure I see why we want to raise a conflict if the content is null.

IMO if we receive null we should throw an exception as this is an invalid parameter.

**Comment by grabbou** [comment]
@SBoudrias well, after giving it a further thought I am kinda confused. This case typically occurs when user chooses a wrong path using in my case Yeoman generator and this file is going to be bundled with the new project that's being generated. 

Actually, I think that checking that using validation after providing value to the input might be the best solution :)

At least we might consider throwing something more interesting than "Cannot read property ... of undefined". WDYT?

**Comment by SBoudrias** [comment]
Well, yeah we should `assert(file)`.

**Pull request closed without merging**